tools/kura_inspector/src/lib.rs

```diff
@@ -1,80 +0,0 @@
-//! General objects independent from executables.
```

Comment:
So we can't use `kura_inspector` as a library now? Was it useless or what?

Reply:
It was not used by anyone at all. It would have been necessary before because kura's storage format was so complicated you would want a library.

Now on the other hand, you don't need a kura_inspector library. Just write the code that inspects the blocks yourself using `BlockStoreTrait`. You get more power, flexibility and the code is still really small.